test/controllers/graphql_controller_test.rb

```diff
@@ -64,6 +64,13 @@ class GraphqlControllerTest < ActionDispatch::IntegrationTest
     assert_equal ["data"], json_response.keys
   end
 
+  test "[if enabled] query public API Namespaces with nested apiResources with non-primitive properties" do
```

Comment:
@Pralish can we add a more in depth test case that tests for: 
1. mime type & object URL 
2. rich text